www/scripts/codecheckerviewer/ListOfBugs.js

```diff
@@ -25,31 +25,41 @@ function (declare, dom, Deferred, ObjectStore, Store, QueryResults, topic,
   BorderContainer, TabContainer, Tooltip, DataGrid, BugViewer, BugFilterView,
   RunHistory, hashHelper, util) {
 
-  function getRunData(runId) {
+  function getRunData(runNames) {
```

Comment:
There is already a very similar function which queries ids instead of names. Should we have all these in separate files?